net.nymtech.NymVPN.yml

```diff
@@ -0,0 +1,48 @@
+id: net.nymtech.NymVPN
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: run.sh
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=/run/nym-vpn.sock:ro
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: nym-vpn
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-app-v0.2.3/nym-vpn_0.2.3_linux_x64
+        sha256: bfc302e582d086c691ac6038bc626ec94f7dae99eb8790ef66785a106a8f9a1c
+        only-arches: [ x86_64 ]
+      - type: file
+        path: net.nymtech.NymVPN.metainfo.xml
+      - type: file
+        path: net.nymtech.NymVPN.desktop
+      - type: script
+        # entry point used by `flatpak run` command
+        dest-filename: run.sh
+        commands:
+          # use `WEBKIT_DISABLE_DMABUF_RENDERER=1` to fix an issue with NVIDIA gpu
+          # https://github.com/nymtech/nym-vpn-client/issues/305
+          - WEBKIT_DISABLE_DMABUF_RENDERER=1 /app/bin/nym-vpn "$@"
```

Comment:
this should only be set if user has nvidia gpu

Reply:
`WEBKIT_DISABLE_DMABUF_RENDERER` seems to be harmless when no nvidia gpu is present on the system

Reply:
And I would like to avoid adding some runtime check in the wrapper script to know if the host system is using nvidia gpu or not.

Reply:
No it's not and you are already using a wrapper. add `if [[ -c /dev/nvidia0 ]]`